prusa/connect/printer/__init__.py

```diff
@@ -682,6 +701,43 @@ def loop(self):
             except Exception:
                 log.exception("Unexpected exception caught in SDK loop!")
 
+    def snapshot_loop(self):
+        """Gets an item Snapshot from queue and sends it"""
+        while self.__running_loop:
+            try:
+                # Get the item to send
+                item = \
+                    self.snapshot_queue.get(timeout=const.TIMESTAMP_PRECISION)
+
+                # Send it
+                headers = self.make_headers(item.timestamp)
```

Comment:
No, i told you, that snapshot will has different headers, make_headers add printer fingerprint to header, but we need camera fingerprint!

`headers` variable can be created in Snapshot.send method - i told you.